packages/vue/src/components/atoms/SfChevron/SfChevron.js

```diff
@@ -0,0 +1,28 @@
+export default {
+  name: "SfChevron",
+  props: {
+    /** Is SfChevron active. Direction of chevron depends on it  */
+    isActive: {
+      type: Boolean,
+      default: false
+    },
+    /** Color of Chevron. Default is black.  */
+    color: {
+      type: String,
+      default: "#000000"
+    },
+    /** Size of Chevron component. Default is 40px.  */
+    size: {
+      type: String,
```

Comment:
In general, color and size customization should be done with CSS modifiers or SCSS vars, not by props.
I think it's a convention for all components...